docs/examples/cheerio_crawler.mdx

```diff
@@ -7,7 +7,9 @@ import CodeBlock from '@theme/CodeBlock';
 import ApiLink from '@site/src/components/ApiLink';
 import CheerioCrawlerSource from '!!raw-loader!./cheerio_crawler.ts';
 
-This example demonstrates how to use <ApiLink to="cheerio-crawler/class/CheerioCrawler">`CheerioCrawler`</ApiLink> to crawl a list of URLs from an external file, load each URL using a plain HTTP request, parse the HTML using the [Cheerio library](https://www.npmjs.com/package/cheerio) and extract some data from it: the page title and all `h1` tags.
+This example demonstrates how to use <ApiLink to="cheerio-crawler/class/CheerioCrawler">`CheerioCrawler`</ApiLink> to crawl a list of URLs from an external
+file, load each URL using a plain HTTP request, parse the HTML using the [Cheerio library](https://www.npmjs.com/package/cheerio) and extract some data from it:
```

Comment:
not sure if those line breaks are coming from some auto formatting. i would rather not see line breaks in text, line wraps should be handled by IDE

Reply:
The ones I did were manually done actually. I can revert them, but then we'll have inconsistencies between docs in different or even the same files, as some have line breaks, and some don't... We should probably really consider setting up a tool to format docs consistently so we don't have issues in the future

Reply:
We already have inconsistencies. We should gradually fix them to the right state = remove line breaks, not add them :]

Not sure if you can force pretties to remove line breaks, I dont think so, its very "inconfigurable" (another thing why I dislike it).

Reply:
Hah, I used to feel the same way about it, then I started using it and I couldn't go back from it (in projects that use it at least) 😄 (it's especially useful when it splits up long lines in a decently readable way, at least if you configure the max width to _not_ be 80 but make it a more reasonable 120-150).

That said, I will revert the line breaks. I still think we should keep them, as it will make seeing diffs easier, but I'll make a PR after this one to bring the rest of them to no line breaks

Reply:
I used to maintain line breaks in docs manually too. In retrospective, it was a very bad decision, the editor width is never the same :] 

Other than the line breaking issue, I would be quite ok with prettier, the one last thing I kinda hate about it is the funky removal of blank lines that is totally inconfigurable (let me just disable that!). They even say "its hard problem so we keep things as they are" or something like that, and then they remove the blank lines inside class definitions etc :D 

I just can't adhere to code style decisions of some "prettier god who thinks knows best". Make things configurable for christ sake :D